plugin-cache/src/npm-download-counts.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var http = require('https');
+
+var Q = require('q');
+
+var packageJson = require('package-json');
+
```

Comment:
I don't know. I logically separate them in my head like that, node specific, 3rd party helpers, actual dependencies. No other reason